src/api-user/src/middleware/auth.js

```diff
@@ -0,0 +1,26 @@
+import jwt from "jsonwebtoken"
+
+export function authMiddleware(req, res, next) {
```

Comment:
Do you realize that this is the signature for `express` middlewares and not `hono` ones?

This means you haven't tested your code.

GPT is not forbidden, but as I mentioned during class, failing to write proper code because of heavy use of AI will result in a big loss of score.

Reply:
You're right, it's technology I haven't had the habit of using. I'd made my api in express mode when I switched to Hono, so I must have gotten confused.